test/integration/user_visits_place_test.rb

```diff
@@ -8,12 +8,12 @@ class UserVisitsPlaceSpec < ActionDispatch::IntegrationTest
     VCR.use_cassette("new place") do
       visit "/"
 
-      page.fill_in 'nav-search', 
+      page.fill_in 'nav-search',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.